pkg/cmdline/cmdline.go

```diff
@@ -165,3 +165,18 @@ func GetInitFlagMap() map[string]string {
 	initflags, _ := Flag("uroot.initflags")
 	return getFlagMap(initflags)
 }
+
+// GetFlagsForModule gets all flags for a designated module
+// and returns them as a space-seperated  string designed to be passed to insmod
+func GetFlagsForModule(name string) string {
+	ret := ""
```

Comment:
generally for a variable to have the "zero value" you just declare it
var ret string

Reply:
Fixed in latest.